controllers/articles.controllers.js

```diff
@@ -8,3 +8,9 @@ exports.getArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getArticles = (req, res, next) => {
+  SelectArticles().then((articles) => {
```

Comment:
We should always avoid variable names starting with capitals unless we're working with classes or constructors! selectArticles would much better adhere to best practice